.github/workflows/build-all.yml

```diff
@@ -107,14 +107,14 @@ jobs:
 
       - name: Publish
         id: push
-        uses: redhat-actions/push-to-registry@v2
+        uses: redhat-actions/push-to-registry@5ed88d269cf581ea9ef6dd6806d01562096bee9c # v2.8
         with:
           image: ${{ steps.build_image.outputs.image }}
           tags: ${{ steps.current-version.outputs.value }} ${{ steps.build_image.outputs.tags }}
           registry: ${{ env.IMAGE_REGISTRY }}
 
       - name: Generate attestation for images
-        uses: actions/attest-build-provenance@v2
+        uses: actions/attest-build-provenance@db473fddc028af60658334401dc6fa3ffd8669fd # v2.3.0
```

Comment:
I think zizmor [auto-trusts GitHub actions (i.e. ones under `actions`)](https://docs.zizmor.sh/audits/#unpinned-uses):

> Official GitHub actions namespaces can be pinned by branch or tag. In other words, actions/checkout@v4 is acceptable, but actions/checkout is not.
> All other actions must be pinned by SHA reference.

so this one can stay I think.

And I think it will maybe make the zizmor check still pass even if you remove the ignore file?

Reply:
> And I think it will maybe make the zizmor check still pass even if you remove the ignore file?

There is still a bowtie action, so the zizmor check will fail if we remove the ignore file

Reply:
I think there we should add ourselves to the trusted action list

Reply:
Which I guess is what you have. So yeah good!

Reply:
Yep, just corrected the config to work exactly like this